extensions/amp-ad/0.1/test/test-amp-ad-3p-impl.js

```diff
@@ -41,6 +41,10 @@ function tests(name) {
         type: '_ping_',
         src: 'testsrc',
       });
+      const link = iframe.doc.createElement('link');
+      link.setAttribute('rel', 'canonical');
+      link.setAttribute('href', 'blah');
+      iframe.doc.head.appendChild(link);
```

Comment:
Add to pass check here
https://github.com/ampproject/amphtml/blob/master/src/document-info.js#L53